graphql.go

```diff
@@ -260,6 +260,18 @@ func (c *Client) Run(ctx context.Context, req *Request, resp interface{}) error
 	return c.runWithJSON(ctx, req, resp)
 }
 
+// Do executes the query with pure HTTP request
+func (c *Client) Do(ctx context.Context, req *http.Request, resp *http.Response) error {
```

Comment:
shouldn't Do return a response?

Reply:
`resp` is a and the response will persist there